modules/i3s/docs/api-reference/i3s-loader.md

```diff
@@ -0,0 +1,114 @@
+
+A loader for loading an [Indexed 3d Scene (I3S) layer](https://github.com/Esri/i3s-spec), and its geometries and textures data.
+
+| Loader         | Characteristic                                                    |
+| -------------- | ----------------------------------------------------------------- |
+| File Format    | [Basis Universal](https://github.com/BinomialLLC/basis_universal) |
+| File Type      | Json, Binary                                                      |
+| File Extension | `.json` (layer), `.bin` (geometries)                              |
+| File Format    | [i3s](https://www.opengeospatial.org/standards/i3s)               |
+| Data Format    | [Data formats](#data-formats)                                     |
+| Supported APIs | `load`, `parse`                                                   |
+
+## Terms
+
+The terms and concepts used in `i3s` module have the corresponding parts [I3S Spec](https://github.com/Esri/i3s-spec/blob/master/format/Indexed%203d%20Scene%20Layer%20Format%20Specification.md).
+
+- `tileset`: I3S Indexed 3D Layer File.
+- `tileHeader`: I3S node file.
+- `tileContent`: I3S node content: geometries, textures, etc.
+
+## Usage
+
+When using loaders.gl's generic [`load`](https://loaders.gl/modules/core/docs/api-reference/load#load) function, user needs explicitly specify `I3Sloader` as auto detect loader from previously [`registered loaders`](https://loaders.gl/modules/core/docs/api-reference/register-loaders) is currently not supported for`I3Sloader`.
+
+```js
+import {I3SLoader} from '@loaders.gl/i3s';
+import {load} from '@loaders.gl/core';
+
+// load tileset
+const tileseturl =
+  'https://tiles.arcgis.com/tiles/z2tnIkrLQ2BRzr6P/arcgis/rest/services/SanFrancisco_Bldgs/SceneServer/layers/0';
+const tileset = await load(tileseturl, I3SLoader);
+
+// load tile with content
+const tileUrl =
+  'https://tiles.arcgis.com/tiles/z2tnIkrLQ2BRzr6P/arcgis/rest/services/SanFrancisco_Bldgs/SceneServer/layers/0/nodes/2';
+const tile = await load(tileUrl, I3SLoader, {loadContent: true});
+
+// load tile content
+// featureUrl is needed to load the tile content
+const tileUrl =
+  'https://tiles.arcgis.com/tiles/z2tnIkrLQ2BRzr6P/arcgis/rest/services/SanFrancisco_Bldgs/SceneServer/layers/0/nodes/2/geometries/0';
+await load(tileUrl, I3SLoader, {tile});
+// load to `tile.content`
+```
+
+## Options
+
+| Option             | Type | Default | Description                                                                                                     |
+| ------------------ | ---- | ------- | --------------------------------------------------------------------------------------------------------------- |
+| `i3s.isTileset`    | Bool | `false` | Whether to load `Tileset` (Layer 3D Index) file. If not specifies, will decide if follow `argis` url convention |
+| `i3s.isTileHeader` | Bool | `false` | Whether to load `TileHeader`(node) file. If not specifies, will decide if follow `argis` url convention         |
```

Comment:
Default is false, but text says "if not specified" - does `false` mean autodetect? This is potentially confusing....